packages/firestore/package.json

```diff
@@ -8,10 +8,11 @@
   "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
   "scripts": {
     "prebuild": "tsc -m es2015 --moduleResolution node scripts/*.ts ",
-    "build": "rollup -c rollup.config.es2017.js && rollup -c rollup.config.es5.js",
+    "build": "rollup -c rollup.config.es2017.js && rollup -c rollup.config.es5.js && yarn build:lite",
     "build:deps": "lerna run --scope @firebase/'{app,firestore}' --include-dependencies build",
     "build:console": "node tools/console.build.js",
-    "build:exp": "rollup -c rollup.config.exp.js",
+    "build:exp": "rollup -c rollup.config.exp.js", 
+    "build:lite": "lerna run --scope @firebase/app-exp --include-dependencies build && rollup -c rollup.config.lite.js",
```

Comment:
You may want to add a new script `build:lite:deps` that builds `@firebase/app-exp` and deps, while keeping this script simply executing rollup. During dev, you execute `build:lite:deps` once to compile all deps, then use `build:lite` for quick firestore rebuild.

Reply:
I assume this is not applicable if I use `devDependencies`? At least the PR builds without it. I am learning by doing here :)